src/library/Uno.Material/Styles/Controls/v2/Button.xaml

```diff
@@ -0,0 +1,798 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:um="using:Uno.Material"
+					xmlns:toolkit="using:Uno.UI.Toolkit"
+					xmlns:not_win="http://uno.ui/not_win"
+					mc:Ignorable="d not_win">
+
+	<!-- Converters -->
+	<um:FromNullToValueConverter x:Key="MaterialNullToTextButtonMarginConverter"
+								 NotNullValue="0,0,2,0"
+								 NullValue="0" />
+
+	<!-- Documented variables -->
+	<Thickness x:Key="M3MaterialButtonMinHeight">40</Thickness>
+	<Thickness x:Key="M3MaterialButtonMinWidth">48</Thickness>
+	<Thickness x:Key="M3MaterialButtonPadding">16,0</Thickness>
+	<Thickness x:Key="M3MaterialTextButtonPadding">12,0</Thickness>
+	<CornerRadius x:Key="M3MaterialButtonCornerRadius">20</CornerRadius>
+	<!-- Text button label based on Label Large Typo -->
+	<x:Double x:Key="M3MaterialButtonFontSize">14</x:Double>
+	<x:Double x:Key="M3MaterialButtonCharacterSpacing">7.143</x:Double>
+
+	<!-- ELEVATED -->
+	<Style x:Key="M3MaterialElevatedButtonStyle"
+		   TargetType="Button">
+
+		<Setter Property="Foreground"
+				Value="{StaticResource PrimaryBrush}" />
+		<Setter Property="Background"
+				Value="{StaticResource SurfaceBrush}" />
+
+		<!--Start: Label Large Typo-->
+		<Setter Property="FontSize"
+				Value="{StaticResource M3MaterialButtonFontSize}" />
+		<Setter Property="FontWeight"
+				Value="Medium" />
+		<Setter Property="FontFamily"
+				Value="{StaticResource MaterialMediumFontFamily}" />
+		<Setter Property="CharacterSpacing"
+				Value="{StaticResource M3MaterialButtonCharacterSpacing}" />
+		<!--End: Label Large Typo-->
```

Comment:
Think it would be worth it to make some M3 resources for each part of the typography style? 
So we would have something like

- `M3MaterialLabelLargeFontSize`
- `M3MaterialLabelLargeFontWeight`
- `M3MaterialLabelLargeFontFamily`
- `M3MaterialLabelLargeCharacterSpacing`

So we'd have a set of values for each text style? Because in Flyouts/Menu it also calls for text not in a TextBlock to be Label Large and I don't want each control to define its own typography values if they can be easily shared.

You can see I'm doing this same thing as you for the Flyouts: https://github.com/unoplatform/Uno.Themes/pull/744

Would be nice to have those resources available to reuse